container/lxd/network.go

```diff
@@ -49,15 +54,23 @@ func (s *Server) EnsureIPv4(netName string) (bool, error) {
 	return modified, nil
 }
 
-func (s *Server) VerifyDefaultBridge(profile *api.Profile, eTag string) error {
+// VerifyNetworkDevice attempts to ensure that there is a network usable by LXD
+// and that there is an eth0 NIC device with said network as its parent.
+// If we are not operating as part of a cluster, an attempt will be made to
+// create a missing eth0 device with the default LXD bridge as its parent.
+// If in clustered mode this attempt is not made - the onus of network
+// configuration falls on the operator.
```

Comment:
We should talk through how we see us operating with spaces and LXD network support. If it isn't terribly hard to do so, being able to create containers based on the networks that LXD defines would be great, and would work whether we are clustered or not.